buildenv/jenkins/omrbuild.groovy

```diff
@@ -243,7 +243,7 @@ def SPECS = [
         'junitPublish' : true
     ],
     'osx_x86-64' : [
-        'label' : 'OSX && x86',
+        'label' : 'compile:osx',
```

Comment:
At some point in the near future we're going to have to distinguish between macOS on x86 and macOS on AArch64.  Using your lettering convention above, does that make this `xosx` ?